src/helpers.ts

```diff
@@ -66,12 +65,12 @@ export function getMonotonicTime() {
 /**
  * Converts the trace events timestamp field from the
  * format -  hrTime[0] * 1e6 + Math.round(hrTime[1] / 1000) to
- * the internal timestamp similar to other event types (journey, step, etc)
+ * the our internal monotonic clock timestamp similar to other event types (journey, step, etc)
```

Comment:
Can we be clear about the unit here? Is it Micros? I'd add it to the function name `convertToMonotonicTimeMicros`

I will say, one thing I love about golang is the `time.Duration` type (everything stored as nanos). Makes things clear!

Reply:
I have updated the comments and also changed the method signatures where required and also kept everything in us without converting them back.